storage-node/src/bin/storage_node.rs

```diff
@@ -1,9 +1,13 @@
+use log::info;
 use storage_node::server::storage_node_serve;
 
 // TODO: Add config here.
 
 #[tokio::main]
 async fn main() {
-    println!("starting storage node server...");
+    let _ = env_logger::builder()
+        .filter_level(log::LevelFilter::Info)
+        .try_init();
```

Comment:
Why do we write `let _ = ...` here instead of directly omitting the `let`?

Reply:
`try_init()` returns a result, which needs to be handled to avoid warnings.